Remove hard-coding of path '/' (#14)
DavidSchinazi/draft-cms-masque-connect-ip: draft-cms-masque-connect-ip.md

datagrams between hosts.

The request-target of a CONNECT-IP request is a URI {{!URI=RFC3986}} which uses
the "https" scheme and a client-specified path. When using HTTP/2
{{!H2=RFC7540}} or later, CONNECT-IP requests use HTTP pseudo-headers with the
following requirements:

* The ":method" pseudo-header field is set to "CONNECT-IP".

* The ":scheme" pseudo-header field is set to "https".

> * The ":path" pseudo-header field is set to the value provided by the client. If
>   none is provided, it is set to "/".

Reviewer: ```suggestion
* The ":path" pseudo-header field is set to the value provided by the client.
  That value MUST NOT be empty.
```

Author: Done.
